$OpenBSD: patch-src_core_qgsapplication_cpp,v 1.9 2016/08/08 17:45:40 landry Exp $
Fix paths to various doc files, installed in share/doc/qgis
per doc/CMakeLists.txt patch
--- src/core/qgsapplication.cpp.orig	Fri Jul  8 14:11:47 2016
+++ src/core/qgsapplication.cpp	Sat Jul 23 12:51:17 2016
@@ -554,18 +554,18 @@ QHash<QString, QString> QgsApplication::uiThemes()
 */
 QString QgsApplication::authorsFilePath()
 {
-  return ABISYM( mPkgDataPath ) + QLatin1String( "/doc/AUTHORS" );
+  return ABISYM( mPkgDataPath ) + QLatin1String( "/../doc/qgis/AUTHORS" );
 }
 /*!
   Returns the path to the contributors file.
 */
 QString QgsApplication::contributorsFilePath()
 {
-  return ABISYM( mPkgDataPath ) + QLatin1String( "/doc/CONTRIBUTORS" );
+  return ABISYM( mPkgDataPath ) + QLatin1String( "/../doc/qgis/CONTRIBUTORS" );
 }
 QString QgsApplication::developersMapFilePath()
 {
-  return ABISYM( mPkgDataPath ) + QLatin1String( "/doc/developersmap.html" );
+  return ABISYM( mPkgDataPath ) + QLatin1String( "/../doc/qgis/developersmap.html" );
 }
 
 /*!
@@ -573,7 +573,7 @@ QString QgsApplication::developersMapFilePath()
 */
 QString QgsApplication::sponsorsFilePath()
 {
-  return ABISYM( mPkgDataPath ) + QLatin1String( "/doc/SPONSORS" );
+  return ABISYM( mPkgDataPath ) + QLatin1String( "/../doc/qgis/SPONSORS" );
 }
 
 /*!
@@ -581,19 +581,19 @@ QString QgsApplication::sponsorsFilePath()
 */
 QString QgsApplication::donorsFilePath()
 {
-  return ABISYM( mPkgDataPath ) + QLatin1String( "/doc/DONORS" );
+  return ABISYM( mPkgDataPath ) + QLatin1String( "/../doc/qgis/DONORS" );
 }
 
 /** Returns the path to the sponsors file. */
 QString QgsApplication::translatorsFilePath()
 {
-  return ABISYM( mPkgDataPath ) + QLatin1String( "/doc/TRANSLATORS" );
+  return ABISYM( mPkgDataPath ) + QLatin1String( "/../doc/qgis/TRANSLATORS" );
 }
 
 /** Returns the path to the licence file. */
 QString QgsApplication::licenceFilePath()
 {
-  return ABISYM( mPkgDataPath ) + QLatin1String( "/doc/LICENSE" );
+  return ABISYM( mPkgDataPath ) + QLatin1String( "/../doc/qgis/LICENSE" );
 }
 
 /*!
